src/handbook/development/releases/planning.md

```diff
@@ -177,14 +151,14 @@ are thus required before the design sprint start.
 
 ## Retrospective/Kick-Off
 
-The day after each FlowFuse release, always a Friday, a meeting is scheduled
+The day after each FlowFuse NPM release, always a Friday, a meeting is scheduled
 by the CTO. This meeting includes 2 parts:
 
  1. A retrospective on the last release. This should capture feedback on what
     went well and what could be improved and generate action items to act on the
     feedback.
  2. Kicking off the next release. The involves:
-    1. The Product Manager describing the themes, goals and priorities of the new release
+    1. The Product Manager describing the themes, goals and priorities of the upcomming issues
```

Comment:
```suggestion
    1. The Product Manager describing the themes, goals and priorities of the upcoming issues
```
In a continuous model, there isn't a break between one release and the next - there is no kick-off. Most team members would already be working on items for the next release. Socialising them on this Friday meeting could be too late. Not sure I have an alternative to suggest - we don't want more meetings. So perhaps its more a forward-looking thing to highlight things coming up beyond the immedate 'up next' items.

Reply:
> So perhaps its more a forward-looking thing to highlight things coming up beyond the immedate 'up next' items.

Yes right